cppcache/integration/test/SslOneWayTest.cpp

```diff
@@ -42,7 +42,10 @@ class SslOneWayTest : public ::testing::Test {
   SslOneWayTest() {
     // You can do set-up work for each test here.
     certificatePassword = std::string("apachegeode");
-    currentWorkingDirectory = boost::filesystem::current_path();
+    serverSslKeysDir = boost::filesystem::path(
+        getFrameworkString(FrameworkVariable::TestServerSslKeysDir));
```

Comment:
See comment above. This could simply be:
`serverSslKeysDir = FrameworkVariable::TestServerSslKeysDir`